src/main/rules/GCI1024/IaC/GCI1024.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Avoid Spring repository call in loop or stream operations",
+  "type": "ECO_CODE",
```

Comment:
type unknown : please check in sonarqube documentaion (custom plugin development) what values we can put here (maybe CODE_SMELL ?)